src/test/features/claim/routes/eligibility/index.ts

```diff
@@ -13,6 +13,8 @@ import { app } from '../../../../../main/app'
 import * as idamServiceMock from '../../../../http-mocks/idam'
 
 const cookieName: string = config.get<string>('session.cookieName')
+const pagePath: string = ClaimPaths.eligibilityStartPage.uri
```

Comment:
Can you please update line 23 to use `pagePath` instead of `ClaimPaths.startPage.uri`?

Reply:
👍